src/cephadm/cephadm

```diff
@@ -325,7 +339,7 @@ class UnauthorizedRegistryError(Error):
 
 
 class Ceph(object):
-    daemons = ('mon', 'mgr', 'mds', 'osd', 'rgw', 'rbd-mirror',
+    daemons = ('mon', 'mgr', 'osd', 'mds', 'rgw', 'rbd-mirror',
```

Comment:
Why the order swap? Does it actually impact the functionality of this patch?

Reply:
In a default/minimal ceph I think it's more likely to have OSD daemons rather than MDS. Anyway the order shouldn't impact the functionality, if a daemon is not found it tests the following.